getting-started/setup-building.rst

```diff
@@ -237,6 +237,14 @@ build the :py:mod:`ctypes` module (without the flag the rest of CPython will
 still build properly).
 
 
+Optimization
+''''''''''''
+
+To get an optimized build of Python, ``configure --enable-optimizations --with-lto``. This sets the default make targets up to enable Profile Guided Optimization (PGO) and may be used to auto-enable Link Time Optimization (LTO) on some platforms. Note that configuration including this option takes time to configure. If you want to learn more about these options, please  see `Configure > Performance options <https://docs.python.org/3/using/configure.html?highlight=lto#performance-options>`_ ::
```

Comment:
We can use intersphinx here, e.g. `` :ref:`performance options <python:performance options>` ``.

Also, we should use an explicit code block with the [``console`` highlighter](https://pygments.org/docs/lexers/#pygments.lexers.shell.BashSessionLexer), e.g.:

```restructuredtext
.. code:: console

   $ ./configure --enable-optimizations --with-lto
```